benchmark/micro/json/RealNest/4_filter_unnested_struct.benchmark

```diff
@@ -0,0 +1,22 @@
+# name: benchmark/micro/json/RealNest/4_filter_unnested_struct.benchmark
+# description: Filter unnested struct Tau
+# group: [RealNest]
+
+name filter_unnested_struct
+group real_nest
+
+require json
+
+load
+CREATE TABLE Run2012B_SingleMu AS SELECT * FROM read_json('duckdb_benchmark_data/hep-adl-ethz-Run2012B_SingleMu/data.jsonl');
+
+run
+SELECT
+    count(*) as total_rows,
+    count(*) FILTER (charge < 0) as negatives,
+    count(*) FILTER (charge = 0) as positives,
+    count(*) FILTER (charge > 0) as neutral,
+    count(*) FILTER (pt - relIso_all < decayMode) as odds,
+    count(*) FILTER (idIsoVLoose != idIsoLoose) as idIsoMatch
+FROM (SELECT UNNEST(Tau, recursive:=true) AS tau FROM Run2012B_SingleMu)
```

Comment:
Is `Tau` a list? Are any of the columns in the `SELECT` clause nested types?

Reply:
`Tau` is a `list` and yes, all the columns are nested here

Reply:
Nice 👍 
So this benchmark contains filters on nested values, and `benchmark/micro/json/RealNest/5_list_filter_on_unnested_structure.benchmark` aggregates them. That's cool.